mscore/editstaff.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>760</width>
-    <height>600</height>
+    <width>895</width>
+    <height>657</height>
```

Comment:
Automatic Qt Designer change. The updated values are the minimum on my computer.

Reply:
you're not forced to commit this part

Reply:
Just trying to fix as soon as possible.

Reply:
Aside from this being unrelated to strings, I don't know if it actually fixes anything. Is it possible that the original 760×600 was chosen for a good reason? Maybe it works better on other OSes or at other DPI settings (or something else)?

Reply:
...